src/python/pants/backend/codegen/thrift/java/java_thrift_library.py

```diff
@@ -84,7 +84,6 @@ def default_java_namespace(self):
   def include_paths(self):
     return self._include_paths
 
-  # TODO(Eric Ayers) As of 2/5/2015 this call is DEPRECATED and should be removed soon
   @property
```

Comment:
Perhaps instead of adding a new call for is_thrift, this could move to a different pattern. Maybe it could look for the class instead via isinstance. It looks like the deprecation plan for the associated `is_*` methods involved types via marker mixins.